### ISSUE TITLE
Enhancement : Adding other examples in readme

### ISSUE THREAD

**Issue by Nariod** [Kudaes/DInvoke_rs#4]
Hello !

Thank you for the repo :)
Could you provide more examples of using the repo ? Especially how to add other API type in the data folder, that would be amazing :)

You can add me on Discord nariod#4621

Cheers !

**Comment by Kudaes** [comment]
Hi @Nariod ,

Exactly what kind of help are you missing? Adding new API and data types is pretty straightforward, you just need to use the following resources to find the desired type / API function and add it to the data crate:

- https://microsoft.github.io/windows-docs-rs/doc/windows/ -> Win32 structures and API headers.
- https://docs.rs/ntapi/0.3.6/ntapi/index.html -> NT structures and API headers.

If you give me a little bit more of context maybe I can give you further assistance.

**Comment by Nariod** [comment]
Hello @Kudaes ,

Thanks for your answer.
I should have closed the issue to be honest, I opened it while trying to understand how and where you added data types in your code to be able to call your APIs, but I did manage to understand it later on :)
I managed to write working Rust droppers using your project (https://github.com/Nariod/Tartocitron/tree/main/DInvoke), so thank you very much for that.

I'm closing the issue.
